Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -131,7 +131,7 @@ extension UserFeedbackUITests {
 
         fillInFields(testMessage, testName, testEmail)
         
-        submit()
+        sendButton.tap()
```

Comment:
`h`: In the implementation of `submit()` you are asserting for `            XCTAssert(widgetButton.waitForExistence(timeout: 1))` if the parameter `expectingError` is set to `false` (it is by default). That means, replacing this call removes the assertion. Please double-check that you actually intended to remove the assertion for the widget button when not using a custom button.

Reply:
Ah, you're right, this is probably why i saw the timing issue come back that i fixed in one of the new test cases.